Anime-App/Feature/Anime/View/AnimeViewController.swift

```diff
@@ -129,12 +134,25 @@ extension AnimeViewController {
         
     }
     
+    func scrollViewDidScroll(_ scrollView: UIScrollView) {
+        let offsetY = scrollView.contentOffset.y
+        let contentHeight = scrollView.contentSize.height
+        
+        if offsetY > contentHeight - scrollView.frame.height {
+            if !isLoading {
+                offset+=5
```

Comment:
When fetching the next set of objects do you not need to use the page also? Since if you reach the limit for the offset i won't be getting the next set of data for page 2. You're going to need to add in extra logic to see if you've reached the offset limit and then go to the next page cos right now you're only fetching the next set of 5 items on the page 1

Did you not test this out by going through all of the pages?